Mathlib/Data/IsROrC/Basic.lean

```diff
@@ -53,6 +53,25 @@ open ComplexConjugate
 
 /--
 This typeclass captures properties shared by ℝ and ℂ, with an API that closely matches that of ℂ.
+
+While the name of the class begins with `Is`,
+this typeclass carries data (all operations common to real and complex numbers),
+not just claims existence of these operations.
+
+This class has exactly two instances: one for `ℝ` and one for `ℂ`.
+In particular, there is no instance for `ULift K`.
```

Comment:
Is there any reason not to add this instance?

Reply:
If you add this instance, then given `[IsROrC K] [NormedSpace K E]`, Lean will search for, e.g., `NormedSpace (ULift (ULift (ULift Real))) E`